VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -65,9 +85,47 @@ def device_list_for_billing(self):
         devices_to_bill = [] #list to store device names to be billed
         device_items = disk_util.get_device_items(None)
         for device_item in device_items :
-            # self.logger.log("Device name : {0} ".format(str(device_item.name)),True)
             if str(device_item.name).startswith("sd"):
                 devices_to_bill.append("/dev/{0}".format(str(device_item.name)))
+            else:
+                self.logger.log("Not adding device {0} as it does not start with sd".format(str(device_item.name)))
+        self.logger.log("Initial billing items {0}".format(devices_to_bill))
+        
+        for file_system in self.file_systems_info:
```

Comment:
Let's add a comment to show how rows are there in file_systems_info: like you did for lsscsi in L101. It will create clarity asto why we are taking item at index 2 to add to root_devices.